python/cugraph/link_analysis/pagerank_wrapper.pyx

```diff
@@ -46,7 +46,7 @@ def pagerank(input_graph, alpha=0.85, personalization=None, max_iter=100, tol=1.
 
     df = cudf.DataFrame()
     df['vertex'] = cudf.Series(np.arange(num_verts, dtype=np.int32))
```

Comment:
Won't this crash with 64 bit vertex IDs as well? (and see the line 37, too).

Reply:
I talked to @ChuckHastings and he said some of legacy algorithm only support int32 vertex ID but those will be removed as we migrate to primitive based implementations

Reply:
I see, but isn't this code for PageRank? @ChuckHastings Do we still have legacy implementation for PageRank? AFAIK, we're using the primitive based implementation for both SG & MG PageRank.

Reply:
If Pagerank is primitive based, can I just do this ?
`[src, dst] = graph_primtypes_wrapper.datatype_cast([input_graph.edgelist.edgelist_df['src'], input_graph.edgelist.edgelist_df['dst']], [np.int32, np.int64])`

`df['vertex'] = cudf.Series(np.arange(num_verts, dtype=src.dtype))`

Reply:
Correct.  The legacy pagerank implementation was deleted a while back.  @jnke2016, your cast as above should work, I believe.